powershell/UpdatesOffline.ps1

```diff
@@ -151,7 +151,7 @@ Import-Module XenServerPSModule
 
 try {
     Write-Host "Connecting to server"
-    Connect-XenServer -Server $Server -UserName $Username -Password $Passwd
+    Connect-XenServer -Server $Server -UserName $Username -Password $Passwd -NoWarnCertificates -NoWarnNewCertificates
```

Comment:
Can we also add that warning comment we have in the other files to remind the user not to do this in production code.